src/panel/context/Explorer/ast/variables.ts

```diff
@@ -0,0 +1,72 @@
+/** Copied from https://github.com/FormidableLabs/urql-exchange-graphcache/blob/master/src/ast/variables.ts */
+
+import { FieldNode, ValueNode, OperationDefinitionNode, Kind } from "graphql";
+
+import { getName } from "./node";
+import { Variables } from "./types";
+
+/** Evaluates a given ValueNode to a JSON value taking vars into account */
+export const evaluateValueNode = (node: ValueNode, vars: Variables): any => {
+  switch (node.kind) {
+    case Kind.NULL:
+      return null;
+    case Kind.INT:
+      return parseInt(node.value, 10);
+    case Kind.FLOAT:
+      return parseFloat(node.value);
+    case Kind.LIST:
+      return node.values.map(v => evaluateValueNode(v, vars));
+    case Kind.OBJECT:
+      return node.fields.reduce((obj, field) => {
+        obj[getName(field)] = evaluateValueNode(field.value, vars);
+        return obj;
+      }, Object.create(null));
+    case Kind.VARIABLE:
+      const varValue = vars[getName(node)];
+      return varValue !== undefined ? varValue : null;
+    default:
+      return node.value;
+  }
+};
+
+/** Evaluates a fields arguments taking vars into account */
+export const getFieldArguments = (
+  node: FieldNode,
+  vars: Variables
+): null | Variables => {
+  if (node.arguments === undefined || node.arguments.length === 0) {
+    return null;
+  }
+
+  return node.arguments.reduce((args, arg) => {
```

Comment:
In graphcache this changed to normalise null nodes https://github.com/FormidableLabs/urql-exchange-graphcache/pull/85/files#diff-1cc3708e82a91cfc26a7682453094fc4R24